client/src/App.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+
+import { TaskAppContainer } from "./components/TaskAppContainer/TaskAppContainer";
+import { Header } from "./components/Header";
+
+import {
+    getItems,
+    postItem,
+    updateItem,
+    deleteItem,
+} from "./services/item_client";
+import "./index.css";
+import "monday-ui-react-core/dist/main.css";
+
+function App() {
+    const [tasks, setTasks] = React.useState([]);
+
+    React.useEffect(() => {
+        getAllTasksFromDB();
+    }, []);
+
+    const getAllTasksFromDB = async () => {
```

Comment:
I think it's better that this and all the methods imported from item_Client should be inside a task list app container and not directly under App.js.
Let's keep App.js clean with only one line of rendering our application container

Reply:
Moved logic to taskAppcontainr